src/test/scala/com/target/data_validator/CliOptionParserSpec.scala

```diff
@@ -0,0 +1,15 @@
+package com.target.data_validator
+
+import org.scalatest.{FunSpec, Matchers}
+
+class CliOptionParserSpec extends FunSpec with Matchers {
+
+  describe("CliOptionParser") {
+    describe("parsing") {
+      it("does not handle var option values with commas") {
+        val args = Array("--vars", "keyA=value1,value2,keyB=value3")
```

Comment:
Could we just use a different delimiter inside the list? is there a use case for list variables?

Reply:
It's unfortunately inflexible. See https://github.com/scopt/scopt/issues/111 as well as the commit message on one of the commits in this PR.